pkg/network/vmispec/network.go

```diff
@@ -43,3 +47,12 @@ func FilterMultusNonDefaultNetworks(networks []v1.Network) []v1.Network {
 	}
 	return multusNetworks
 }
+
+func LookUpDefaultNetwork(networks []v1.Network) (*v1.Network, error) {
+	for _, network := range networks {
+		if network.Multus == nil || network.Multus.Default {
+			return &network, nil
```

Comment:
It is discouraged to pass the address of the value which comes from the range.
You better return the original entity address: `&networks[i]`.

Reply:
totally! ! !